shanoir-ng-datasets/src/main/java/org/shanoir/ng/importer/strategies/dataset/CtDatasetStrategy.java

```diff
@@ -0,0 +1,149 @@
+ */
+
+package org.shanoir.ng.importer.strategies.dataset;
+
+import org.dcm4che3.data.Attributes;
+import org.shanoir.ng.dataset.modality.CtDataset;
+import org.shanoir.ng.dataset.model.CardinalityOfRelatedSubjects;
+import org.shanoir.ng.dataset.model.DatasetExpression;
+import org.shanoir.ng.dataset.model.DatasetMetadata;
+import org.shanoir.ng.dataset.model.DatasetModalityType;
+import org.shanoir.ng.dataset.model.ProcessedDatasetType;
+import org.shanoir.ng.dicom.DicomProcessing;
+import org.shanoir.ng.importer.dto.Dataset;
+import org.shanoir.ng.importer.dto.DatasetsWrapper;
+import org.shanoir.ng.importer.dto.ExpressionFormat;
+import org.shanoir.ng.importer.dto.ImportJob;
+import org.shanoir.ng.importer.dto.Serie;
+import org.shanoir.ng.importer.strategies.datasetexpression.DatasetExpressionContext;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CtDatasetStrategy implements DatasetStrategy<CtDataset> {
+
+	@Autowired
+	DicomProcessing dicomProcessing;
+	
+	@Autowired
+	DatasetExpressionContext datasetExpressionContext;
+	
+	@Override
+	public DatasetsWrapper<CtDataset> generateDatasetsForSerie(Attributes dicomAttributes, Serie serie,
+			ImportJob importJob) throws Exception {
+		
+		DatasetsWrapper<CtDataset> datasetWrapper = new DatasetsWrapper<>();
+		/**
+		 * retrieve number of dataset in current serie if Number of dataset > 1 then
+		 * each dataset will be named with an int at the end of the name. else the is
+		 * only one dataset => no need for extension.
+		 */
+		int datasetIndex;
+		if (serie.getDatasets().size() > 1) {
+			datasetIndex = 1;
+		} else {
+			datasetIndex = -1;
+		}
+
+		// TODO ATO : implement MrDAtasetAcquisitionHome.createMrDataset (issue by
+		// createMrDatasetAcquisitionFromDicom()
+		for (Dataset anyDataset : serie.getDatasets()) {
+			// TODO ATO : implement line 350 - 372 MrDAtasetAcquisitionHome.createMrDataset
+			CtDataset dataset = new CtDataset();
+			dataset = generateSingleDataset(dicomAttributes, serie, anyDataset, datasetIndex, importJob);
+			datasetWrapper.getDatasets().add(dataset);
+			datasetIndex++;
+		}
+
+		return datasetWrapper;
+
+	}
+
+	/* (non-Javadoc)
+	 * @see org.shanoir.ng.dataset.modality.DatasetStrategy#generateSingleMrDataset(org.dcm4che3.data.Attributes, org.shanoir.ng.importer.dto.Serie, org.shanoir.ng.importer.dto.Dataset, int, org.shanoir.ng.importer.dto.ImportJob)
+	 */
+	@Override
+	public CtDataset generateSingleDataset(Attributes dicomAttributes, Serie serie, Dataset dataset, int datasetIndex,
+			ImportJob importJob) throws Exception {
+		CtDataset ctDataset = new CtDataset();
+		ctDataset.setCreationDate(serie.getSeriesDate());
+		final String serieDescription = serie.getSeriesDescription();
+
+		DatasetMetadata datasetMetadata = new DatasetMetadata();
+		ctDataset.setOriginMetadata(datasetMetadata);
```

Comment:
Maybe make this call at the end of the method and do 
`datasetMetadata.setName(computeDatasetName(serieDescription, datasetIndex));`
instead of
`ctDataset.getOriginMetadata().setName(computeDatasetName(serieDescription, datasetIndex));`

Reply:
> > Mais je ne retrouve pas tes logs qui viennent visiblement de l'import.
> 
> Les logs venaient de 'datasets' au moment de la création des DatasetAcquisitions

Mais le dataset est quand même importé ?